videointelligence/google/cloud/videointelligence_v1beta1/gapic/video_intelligence_service_client.py

```diff
@@ -88,6 +88,8 @@ def __init__(self,
                 client library metrics. Ultimately serializes to a string
                 (e.g. 'foo/1.2.3 bar/3.14.1'). This argument should be
                 considered private.
+
+        Returns: VideoIntelligenceServiceClient
```

Comment:
@lukesneeringer I thought you were anti-"returns in `__init__`"?

Reply:
I am. :-) I could have sworn I removed that.

Reply:
Yeah, I definitely got a old version or something somewhere. Fixing.

Reply:
Canary in the GitHub mine

Reply:
Finally got new stuff built and fix pushed.